### PR TITLE
Better documentation and fixes for #6, #8 and #11

### PR DESCRIPTION
**Pull request by ascott1**
- documents the use of individual icons with topdoc
- Replaces the `-alt` suffix with `-round`
- Adds the EOF Topdoc comment

### CODE REVIEW

File: demo/static/css/main.lt-ie9.min.css [hunk 1]
```diff
@@ -65,7 +53,7 @@
         <i class="cf-icon cf-icon-share"></i> Standard Icon
     - name: Alt Icon Use
```
Comment (himedlooff) [inline demo/static/css/main.lt-ie9.min.css]: Should this say "Rounded Icon Use"?

**Comment by ascott1** [comment]
Sorry, no more pushes until this is reviewed :wink:

**Comment by Scotchester** [comment]
This is good to go. We need to do some work to make the docs/demo styling a bit easier on the eyes, but that's more of a cf-component-demo problem. Thanks!

**Comment by himedlooff** [comment]
There is already an open issue for the code_examples template inspired by this: https://github.com/cfpb/cf-component-demo/issues/10

I think the demo repo needs to be as lean as possible and only include the component-related CSS for testing purposes.

**Pull request merged**